src/LibraryManager/Providers/jsDelivr/JsDelivrCatalog.cs

```diff
@@ -231,6 +239,13 @@ public async Task<CompletionSet> GetLibraryCompletionSetAsync(string libraryName
 
                         completions.Add(completionItem);
                     }
+
+                    // support @latest version
+                    completions.Add(new CompletionItem
```

Comment:
Does it end up at the very bottom? Seems like it should be at the very top, no?

Reply:
It does go at the bottom, intentionally. Putting it first is like recommending it the most.  I wouldn't recommend using the `latest` version tag very often - only in experimental/prototypal situations.  It could bring in new dependencies randomly (i.e. as soon as they're released), which could include breaking changes (e.g. bootstrap 3.x -> 4.x was a big breaking change).  It's considerably a best-practice to always explicitly manage your dependency versions.

If we get feedback requesting it higher, then we can change the ordering.